lib/rules/disallow-space-after-keywords.js

```diff
@@ -65,8 +67,8 @@ module.exports.prototype = {
         file.iterateTokensByTypeAndValue('Keyword', this._keywords, function(token) {
             var nextToken = file.getNextToken(token);
 
-            // Make an exception if the next token is also a keyword (a space would be required).
-            if (nextToken.type === 'Keyword') {
+            // Make an exception if the next token is not a Punctuator such as a Keyword or Identifier
+            if (nextToken.type !== 'Punctuator') {
```

Comment:
The previous fix for `else if` was adding `nextToken.type === 'Keyword'`

I was going to check `nextToken.type == 'Identifier'` but I figured maybe this could be better?

@mikesherov